recipes/h2/meta.yaml

```diff
@@ -0,0 +1,42 @@
+{% set name = "h2" %}
+{% set version = "3.0.1" %}
+{% set sha256 = "b2962f883fa392a23cbfcc4ad03c335bcc661be0cf9627657b589f0df2206e64" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record=record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+  run:
+    - python
+    - hyperframe    >=5.1.0,<6
+    - hpack         >=2.3,<4
```

Comment:
Looks like you missed [the version-based dependency on `enum34`](https://github.com/python-hyper/hyper-h2/blob/c28ea98c720765b56c73fc511aa9ad1d81d58ea0/setup.py#L69): in conda-land, this should be  <code>- enum34 >=1.1.6,<2 &nbsp;# [py2k]</code>.